torchmetrics/classification/accuracy.py

```diff
@@ -269,6 +272,8 @@ def compute(self) -> Tensor:
         """
         Computes accuracy based on inputs passed in to ``update`` previously.
         """
+        if not self.mode:
```

Comment:
Is this even possible based on the conditions before?

Reply:
yes if you call compute without any update